nav/backplanes/backplanes_bodies.py

```
@@ -0,0 +1,153 @@
+                    # Try direct key
+                    mask = mask_map.get(body_name, None)
+                if mask is not None:
+                    sub_mask = mask[v0:v1 + 1, u0:u1 + 1].astype(bool)
+                if sub_mask is None:
+                    try:
+                        # Build robust name list for matching
+                        order_names = [str(n).upper() for n in snapshot.sim_body_order_near_to_far]
+                        body_idx = order_names.index(str(body_name).upper()) + 1
+                        sim_map = snapshot.sim_body_index_map
+                        sub_mask = (sim_map[v0:v1 + 1, u0:u1 + 1] == body_idx)
+                    except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)